components/profile_picture.jsx

```diff
@@ -51,7 +52,7 @@ export default class ProfilePicture extends React.PureComponent {
                         />
                     }
                 >
-                    <span className='status-wrapper'>
+                    <span className={`status-wrapper  ${this.props.helperClass}`}>
```

Comment:
More nitpicking, this time about whitespace
```suggestion
                    <span className={`status-wrapper ${this.props.helperClass}`}>
```

Reply:
Totally fine : )